generate-images.js

```diff
@@ -0,0 +1,27 @@
+const fs = require('fs')
+const glob = require('glob')
+const puppeteer = require('puppeteer')
+const contents = require('./contents.js')
+
+glob('./cards/**.png', async (err, filePaths) => {
```

Comment:
we may want to allow a write location to be specified as a parameter of this script